### ISSUE TITLE
Replace/Clear a destination file by default

### ISSUE THREAD

**Issue by lukeed** [lukeed/taskr#121]
Instead of preceding each task with `this.clear(paths.dest)`, what if Fly either (1) did it automatically on a per-file basis, or (2) just replaced/overwrote the destination file?

This is expected behavior, as Gulp/Grunt/Webpack have all done this.

And, as I'm sure you know, **not** preceding the task with `clear()` results in appending the new content to the file; eg:

``` javascript
'use strict';
console.log('this is first save');
//# sourceMappingURL=app.js.map
'use strict';
console.log('this is SECOND save');
//# sourceMappingURL=app.js.map
```

Seems strange to me, but there may be practical use cases of this ...? /shrug

**Comment by ghost** [comment]
@lukeed You found my little dirty secret. This is one of the few remaining TODOs since the early days. 

(1) or (2) either solution is acceptable. How keen would you be on submitting a patch for this?

**Comment by lukeed** [comment]
2 would be the way to go IMO. And I'll try, will be traveling.

**Comment by lukeed** [comment]
@bucaran I fixed this :raised_hands:  Will open a PR when #122 gets resolved

**Comment by lukeed** [comment]
Closing. Solved by #126
